addons/Rules/Tweek.Drivers.Rules.FileSystem/FileSystemRulesDriver.cs

```diff
@@ -0,0 +1,36 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Tweek.Engine.Drivers.Rules;
+
+namespace Tweek.Drivers.Rules.FileSystem
+{
+  public class FileSystemRulesDriver : IRulesDriver
```

Comment:
Not sure there's a need for a version file in FileSystemRulesDriver (it could be a fixed value), the version is designed to work with publishing, in a StandAlone I'm not sure there's a case for that.

Reply:
👍 

I'll turn the version into a fixed value as you suggested (can't remove the `GetVersion` function since we implement the `IRulesDriver` interface) and change the `DirectoryPath` env var to `FilePath` (so the fixed value from the `GetVersion` function will not actually be used, the `GetRuleset` function will always return the file from the env var value)

Reply:
Done here https://github.com/Soluto/tweek/pull/1336